src/lib/viewers/image/__tests__/ImageViewer-test.js

```diff
@@ -421,26 +421,45 @@ describe('lib/viewers/image/ImageViewer', () => {
     });
 
     describe('loadUIReact()', () => {
-        test('should create controls root and render the react controls', () => {
-            image.options.useReactControls = true;
-            image.loadUIReact();
-
-            expect(image.controls).toBeInstanceOf(ControlsRoot);
-            expect(image.controls.render).toBeCalledWith(
-                <ImageControls
-                    annotationMode="none"
-                    hasHighlight={false}
-                    hasRegion={false}
-                    onAnnotationModeClick={image.handleAnnotationControlsClick}
-                    onAnnotationModeEscape={image.handleAnnotationControlsEscape}
-                    onFullscreenToggle={image.toggleFullscreen}
-                    onRotateLeft={image.rotateLeft}
-                    onZoomIn={image.zoomIn}
-                    onZoomOut={image.zoomOut}
-                    scale={1}
-                />,
-            );
-        });
+        test.each`
+            areNewAnnotationsEnabled | hasAnnotationCreatePermission | hasDrawing | hasRegion | showAnnotationsDrawingCreate
+            ${false}                 | ${false}                      | ${false}   | ${false}  | ${false}
```

Comment:
It seems like we should have a line for each permutation of the input values.

Reply:
Yep, will add the remaining permutations.